x/gov/README.md

```diff
@@ -116,27 +120,28 @@ proposal is finalized (passed or rejected).
 #### Deposit refund and burn
 
 When a proposal is finalized, the coins from the deposit are either refunded or burned
-according to the final tally of the proposal:
+according to the final tally of the proposal. In Heimdall, burn is not enabled, hence all the deposits will be refunded.
 
 * If the proposal is approved or rejected but *not* vetoed, each deposit will be
   automatically refunded to its respective depositor (transferred from the governance
   `ModuleAccount`).
-* When the proposal is vetoed with greater than 1/3, deposits will be burned from the
+* When the proposal is vetoed with greater than 1/3, deposits will be refunded from the
   governance `ModuleAccount` and the proposal information along with its deposit
   information will be removed from state.
-* All refunded or burned deposits are removed from the state. Events are issued when
-  burning or refunding a deposit.
+* All refunded deposits are removed from the state. Events are issued when
+  refunding a deposit.
 
 ### Vote
 
 #### Participants
 
 *Participants* are users that have the right to vote on proposals. On the
-Cosmos Hub, participants are bonded Atom holders. Unbonded Atom holders and
-other users do not get the right to participate in governance. However, they
-can submit and deposit on proposals.
+Polygon PoS network, participants are validators. Other holders and users do not get the right to participate in governance. 
+However, they can submit and deposit on proposals.
+
+// TODO HV2: if we don't want non-validators to submit/deposit on proposals, we would need to add an additional check then on gov proposal submission, to validate the proposer is in the active set
```

Comment:
Wasn't that the role of the _ValidatorID_ field that was added in HeimdallV1 to many of the gov messages? (see e.g., definition of struct `MsgSubmitProposal` [here](https://github.com/Raneet10/cosmos-sdk/pull/4/files#diff-7cb7d2e6bb850859f63e49884361a66cfbb63c2b082b14f8ebc699d2e86b243a)).
Was it a conscious decision not to forward-port those added fields?

Reply:
Never mind, I see below it was conscious as _ValidatorID_ is redundant with address. And now understand your comment about the "extra check". I totally agree.

Reply:
Yes, the only impact I could see is in L1 contracts where we use validator IDs iirc. Such contracts should anyway provide an utility function to convert from ID to address and vice-versa. I already took a note to discuss this with smart contracts team.

Reply:
👍 resolved